### DISCUSSION TITLE
Device has high linkquality (lqi) but does not respond

### DISCUSSION THREAD

**Discussion by hocomadvies** [Koenkk/zigbee2mqtt#13354] [Devices]
Device recognised as Tuya TS011F_plug_1 (has different look than the image in Zigbee2mqtt list). It is a white label product.

Device has high linkquality (lqi) but does often not respond when on / off command is published. Not very stable compared with other devices. How to improve the working? Other config? 

<img width="1127" alt="Schermafbeelding 2022-08-01 om 14 39 09" src="https://user-images.githubusercontent.com/10847395/182152448-f6336d44-a453-4b33-b1d8-a7fa1737ab6b.png">
<img width="1137" alt="Schermafbeelding 2022-08-01 om 14 39 38" src="https://user-images.githubusercontent.com/10847395/182152455-aa60bc95-4c73-4edd-b56c-fcd0dc48ebab.png">
<img width="629" alt="Schermafbeelding 2022-08-01 om 14 40 18" src="https://user-images.githubusercontent.com/10847395/182152465-d0c1d0fb-4396-47ac-ba64-466c4d830a35.png">

**Comment by hocomadvies** [comment]
Any ideas to get this device working properly?

**Comment by Koenkk** [comment]
I expect that the connection is unstable, the LQI you see is of the last hop. The network map can be used to further diagnose link issues.
